contracts/hts-precompile/README.md

```diff
@@ -2,75 +2,73 @@
-| `updateTokenInfo`                         | `0x18370d34`  | [0.30](https://docs.hedera.com/hedera/networks/release-notes/services#v0.30)  | [HIP 514](https://hips.hedera.com/hip/hip-514)    | `updateTokenInfo(address token, HederaToken hederaToken)` | v2. |
-| `updateTokenInfo`                         | `0x7d305cfa`  | [0.32](https://docs.hedera.com/hedera/networks/release-notes/services#v0.32)  | [HIP 514](https://hips.hedera.com/hip/hip-514)    | `updateTokenInfo(address token, HederaToken hederaToken)` | v3. |
-| `updateTokenKeys`                         | `0x6fc3cbaf`  | [0.30](https://docs.hedera.com/hedera/networks/release-notes/services#v0.30)  | [HIP 514](https://hips.hedera.com/hip/hip-514)    | `updateTokenKeys(address token, Expiry expiry)` |
-| `wipeTokenAccount`                        | `0x9790686d`  | [0.29](https://docs.hedera.com/hedera/networks/release-notes/services#v0.29)  | [HIP 514](https://hips.hedera.com/hip/hip-514)    | `wipeTokenAccount(address token, address account, uint32 amount)` |
-| `wipeTokenAccount`                        | `0xefef57f9`  | [0.32](https://docs.hedera.com/hedera/networks/release-notes/services#v0.32)  | [HIP 514](https://hips.hedera.com/hip/hip-514)    | `wipeTokenAccount(address token, address account, int64 amount)` |  v2. Update amount to int64 | 
-| `wipeTokenAccountNFT`                     | `0xf7f38e26`  | [0.29](https://docs.hedera.com/hedera/networks/release-notes/services#v0.29)  | [HIP 514](https://hips.hedera.com/hip/hip-514)    | `wipeTokenAccountNFT(address token, address account, int64[] serialNumbers)` |
-
-
+| Function Name                          | hash         | Consensus Node Release Version                                               | HIP                                            | Method Interface                                                                                                                                                              | Comments                                                       |
+| -------------------------------------- | ------------ | ---------------------------------------------------------------------------- | ---------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------- |
+| `allowance`                            | `0x927da105` | [0.26](https://docs.hedera.com/hedera/networks/release-notes/services#v0.26) | [HIP 514](https://hips.hedera.com/hip/hip-514) | `allowance(address token, address owner, address spender)`                                                                                                                    |
```

Comment:
Unfortunately this particular change (prettier I assume?) doesn't help IMO - makes it difficult for anyone to fix any of this in an editor and is entirely useless (IMO again) as the point of a markup notation is that you only view it _rendered_ and so what it looks like raw doesn't _have_ to imitate what it would look like rendered.